### ISSUE TITLE
multiple daemon running in single machine

### ISSUE THREAD

**Issue by amrouchk** [ipfs/java-ipfs-api#121]
I would like to know if there is a possibility to run multiple daemons on same machine,? Ive a scenario in which I need to run multiples ipfs nodes on the same  machine with different daemons , I read this proposition in this link https://stackoverflow.com/questions/40180171/how-to-run-several-ipfs-nodes-on-a-single-machine it was exactly what I am looking for but though I am running ipfs on  windows I got issues on initializing ipfs and run multiple daemons. 
following that proposed solution in the previous link ,  I manually created different folders ipfs2,ipfs3,ipfs4 etc.. and added them to the path of windows, then I tried to execute the command
C:\ipfs2>ipfs init and got this issue:
Error: cannot acquire lock: creation of lock C:\Users\amar.ipfs\repo.lock failed: The process cannot access the file because it is being used by another process.
appreciate your suggestions and helps in advance
Regards

**Comment by ianopolous** [comment]
This is an issue for ipfs itself. This repo is just the http client. 

If you want to run multiple ipfs instances on the same machine you need to set their api, gateway and swarm ports to be different for each instance. You can do this with the ipfs config command. The problem you are seeing though is because you also need to set the IPFS_PATH when running a an ipfs command.

**Comment by amrouchk** [comment]
Thanks so much for the precious suggestion.
it was exactly what I did , and runs successfully 
since Im using windows: I run the following 
set IPFS_PATH=C:\Users\Myuser.ipfs2
then I changed the config by different address.
Ive only one question... if I close the cmd prompt how can I run the second ipfs Ive just created. it seems if I close it I should do the same thing again and again...
appreciate your help..
